### ISSUE TITLE
FTA Notepad flashing `DISCONNECT`

### ISSUE THREAD

**Issue by DarkMetalMouse** [FIRST-Tech-Challenge/scorekeeper#704]
**Describe the bug**
The red `DISCONNECTED` text constantly appears and disappears, many times per second

**To Reproduce**
1. Go to FTA Notepad
2. you should see the `DISCONNECTED` text flash

**Expected behavior**
The FTA Notepad should connect, as it workes on othee the devices.

**Screenshots**

https://github.com/FIRST-Tech-Challenge/scorekeeper/assets/51059131/0dc1345b-b985-4804-b524-911a47e895c1

**Device**
 - OS: iPad 2 mini, iPadOS 12.5.*
 - Browser: chrome (also happens on safari)
 - Version: 91.0.4472.80 (also happens on every lower version we tested

**Additional context**
This behavior is consistent with all of our iPad 2 mini devices and isn't reproducible anywhere else (tested on android, windows, macOS, iOS).

I captured some of the network traffic and the part that seems to be wrong is this request:
```
ws://10.10.3.3/stream/fta/?code=[object%20CloseEvent]
```
The code is sending an `Event` object which looks like this:
<img width="1446" alt="Screenshot 2024-02-19 at 15 44 44" src="https://github.com/FIRST-Tech-Challenge/scorekeeper/assets/51059131/3baea958-b6d8-44c2-8614-6f636882419a">
(Screenshot from Safari Web Inspector)
I looked at the same request on a different device, and it was passing a string of the event name to the code parameter:
```
ws://10.10.3.3/stream/fta/?code=ilshalq1
```
I tried automatically replacing the `[object%20CloseEvent]` with the event's name using a proxy and it got further with the transactions, but still failed.

I would appreciate any input on this as it is blocking us from using FTA notepad and we want to avoid connecting the CSA/FTA(A)'s phones to the arena, thanks!

**Comment by AlexOberfranc** [comment]
the scoring system might not support the version of iOS and chrome

**Comment by maths22** [comment]
This should be resolved in v5.2.14
